alex/applications/PublicTransportInfoCS/directions.py

```diff
@@ -482,9 +486,117 @@ def search_stop(self, stop_mask, city=None, max_count=0, skip_count=0):
             TTLANG.ENGLISH # language
         )
 
+    def search_train_station(self, stop_mask, max_count=0, skip_count=0):
```

Comment:
Is this here because we don't know which stations are for trains from our database?